gulpfile.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+/**
+ * @file
+ * This file contains gulp configurations for setting up SASS with feature of
+ * importing Shoreditch Partials and minifying the css file to .min.css
+ *
+ * Tasks
+ * default : Runs SASS task
+ * sass: Compiles civicase.scss under scss folder to CSS counterpart
+ * watch: Watches for scss file changes and run sass task
+ */
+const gulp = require('gulp');
```

Comment:
Sorry I forgot to mention. Lets not use ES6 yet, as we are not using ES6 anywhere else. We definitely want to switch to ES6 in future, then we would switch both the angular and gulpfile. Till then lets use ES5 only. Please fix this for the whole file.

Reply:
Done.